clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1118,7 +1118,7 @@ public void testManualAssignmentChangeWithAutoCommitEnabled() {
 
         ConsumerRecords<String, String> records = consumer.poll(Duration.ofMillis(1));
         assertEquals(1, records.count());
-        assertEquals(11L, consumer.position(tp0));
+        assertEquals(11L, consumer.position(tp0, Duration.ofSeconds(2)));
```

Comment:
Unless there's a good reason for these changes, can you revert them?

Reply:
Hi @hachikuji, I reverted all test changes.